### PR TITLE
Make the coagulation tank parallelable (and a bunch of other things)

### PR DESCRIPTION
**Pull request by MCTian-mi**
- Fixed hardcoded text in stacks display
- Made the coagulation tank parallelable
- Added tooltip for the coagulation tank illustrating the parallel mechanism (and a joke)
- Added independent + and - button textures for steam and primitive machines
- Added independent fluid slot texture for primitive machines

### CODE REVIEW

File: src/main/resources/assets/susy/lang/en_us.lang [hunk 1]
```diff
@@ -231,14 +233,14 @@ gregtech.machine.advanced_arc_furnace.name=Advanced Arc Furnace
 gregtech.machine.clarifier.name=Clarifier
 gregtech.machine.evaporation_pool.name=Evaporation Pool
 gregtech.machine.flare_stack.name=Flare Stack
-gregtech.machine.flare_stack.height=Stack Hight: %s
+gregtech.machine.flare_stack.height=Stack Hight: %s blocks
```
Comment (last8kings) [inline src/main/resources/assets/susy/lang/en_us.lang:236]: should be height

File: src/main/resources/assets/susy/lang/en_us.lang [hunk 2]
```diff
@@ -231,14 +233,14 @@ gregtech.machine.advanced_arc_furnace.name=Advanced Arc Furnace
...
 gregtech.machine.flare_stack.rate=Flaring Rate: %s
 gregtech.machine.flare_stack.rate_hover=Flaring rate multiplier set by the height of the flare stack.
 gregtech.machine.flare_stack.tooltip=Industrial lung cancer propagator.
 gregtech.machine.flare_stack.tooltip.1=§6Base Flaring Rate:§7 §f%d L every 10 ticks
 gregtech.machine.flare_stack.tooltip.2=Doubles its speed for each block taller than 5
 gregtech.machine.smoke_stack.name=Smoke Stack
-gregtech.machine.smoke_stack.height=Stack Hight: %s
+gregtech.machine.smoke_stack.height=Stack Hight: %s blocks
```
Comment (last8kings) [inline src/main/resources/assets/susy/lang/en_us.lang:243]: should be height

**Comment by last8kings** [comment]
@bruberu besides those typos the PR should be fine

**Comment by loxoDev** [comment]
Looks good to me. I will merge this now

**Review by loxoDev — APPROVED** [review]

**Pull request merged**